clarifai.go

```diff
@@ -37,7 +37,7 @@ func NewClarifaiClient() *ClarifaiClient {
 }
 
 // Check whether the client still has access rights to Clarifai
-func (client *ClarifaiClient) isAccess() bool {
+func isCLFAccessible(client *ClarifaiClient) bool {
```

Comment:
Sure, I'm good with following your approach. In this case, there is indeed not much difference 😄 
And I agree that for big projects with many packages or APIs that will be exposed externally, encapsulate the functions as methods under structs is easier for the consumers to use. But for simple web APIs or things that don't have direct interaction with client side, is it still a good practice? I'm not trying to figure out right and wrong. Just want to have a better understanding over this interesting question:
"When should we use methods and when to use functions in Golang?" Technically, making a function as a pointer method under a struct has exactly the same effects as passing the struct pointer to a function. But in which cases do we prefer one over another? I've searched online but haven't found good answers yet. One answer mentioned that they are different in a sense that functions with struct pointers as arguments are more generic as we can pass any structs sharing the same interface to it, while method is only belong to one struct. 
Any insights on this? If not, maybe we can ask in the go channel?